soh/src/code/z_demo.c

```diff
@@ -509,7 +509,54 @@ void Cutscene_Command_Terminator(GlobalContext* globalCtx, CutsceneContext* csCt
         temp = 1;
     }
 
-    if ((csCtx->frames == cmd->startFrame) || (temp != 0) || ((csCtx->frames > 20) && (randoCsSkip || debugCsSkip))) {
+    bool playCutscene = false;
+    if (!CVar_GetS32("gCreditsFix", 1) && (cmd->startFrame == csCtx->frames)) {
+        playCutscene = true;
+    } else if (CVar_GetS32("gCreditsFix", 1)) {
+        u16 delay = 0;
+        
+        // HACK:  Align visual timing with audio curing credits sequence
```

Comment:
Literally not important at all but I think there's a small typo in the comment

Reply:
Oh thanks, I was looking for an excuse to remove an extra newline I didn't need too!